From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: lexikiq <noellekiq@gmail.com>
Date: Thu, 1 Jul 2021 23:23:10 -0400
Subject: [PATCH] Fix reobfuscation

paperweight currently has an issue that prevents it from re-obfuscating anything in
Paper's manual remappings. This patch temporarily fixes important re-obfuscation failures.

diff --git a/src/main/java/net/minecraft/world/effect/MobEffect.java b/src/main/java/net/minecraft/world/effect/MobEffect.java
index 79e036d79dec2ec4404baf02c23ba5ccad20cdce..5aa46ba02c5b94c04ec3d868723b12e0352301e3 100644
--- a/src/main/java/net/minecraft/world/effect/MobEffect.java
+++ b/src/main/java/net/minecraft/world/effect/MobEffect.java
@@ -37,6 +37,13 @@ public class MobEffect {
         return (MobEffect) Registry.MOB_EFFECT.byId(rawId);
     }
 
+    // Parchment start
+    @Nullable
+    public static MobEffect fromId(int rawId) {
+        return byId(rawId);
+    }
+    // Parchment end
+
     public static int getId(MobEffect type) {
         return Registry.MOB_EFFECT.getId(type);
     }
